tbx/project_styleguide/templates/patterns/styleguide/components/components.yaml

```diff
@@ -1,5 +1,23 @@
+context:
+  blockquote_person_data:
+    quote: We spent significant time designing CHI's website navigation to ensure easy access to key content on each page, focusing on usability for all devices. This includes details about staff, nearby hospitals, and CHI's services
+    author_image: True
+    name: Jeth Ordeniza
+    role: UI Designer
+    company: Torchbox
+
+  blockquote_client_data:
+    quote: Torchbox's collaboration has been delightful, significantly elevating NHS.UK team's appreciation of SEO. They're diligent, eager, and proactive, consistently delivering optimal solutions for any challenge
+    company_logo: True
+    name: Theo Smart
+    role: Head of Digital
+    company: NHS UK
+
 tags:
   image:
     avatar width-143 format-webp class="avatar__image":
       raw: |
         <img src="https://source.unsplash.com/143x143?face" width="143" height="143" alt="Some alt" class="avatar__image">
+    value.company_logo width-110 format-webp class="blockquote__company-logo":
```

Comment:
Nitpick: it would be good to use a non-square image here as many logos are more landscape in format.

Reply:
Updated in https://github.com/torchbox/torchbox.com/pull/26/commits/b13842535d328a7998e2e79dbc3d316760867e9b